en/gps_compass/rtk_gps_fem_mini2.md

```diff
@@ -0,0 +1,61 @@
+# Femtones MINI2 receiver
+
+[MINI2 Receiver](http://www.femtomes.com) supports the absolute accuracy positioning of Global Navigation Satellite System (GNSS).
+This delivers the quickest and most reliable RTK initializations for centimeter level positioning.
+Mainly facing to applications of high-precision positioning, navigation and mapping, etc. 
+Customers benefit from the Ethernet connectivity available on the board, allowing high speed data transfer and configuration viastandard web browsers.
+Serial port, USB and CAN are also supported.
+
+<img src="http://www.femtomes.com/image/PX4/MINI_II_Receiver.jpg" width="500px" title="mini2" />
+
+## Required Firmware Options
+
+The following firmware options need to be selected when buying the device:
+
+- 5Hz 10Hz 20Hz 
+- INS 
+- HEADING
+- OBS
+- RTK
+- BASE 
+
+## Purchase
+
+* [http://www.femtomes.com](http://www.femtomes.com)
+* [sales@femtomes.com](mailto:sales@femtomes.com)
+* TEL: +86-10-53779838
+
+## Functional Ports
+
+<img src="http://www.femtomes.com/image/PX4/MINI_II_1.jpg" width="500px" title="mini2port" />
+
+## Wiring and Connections
+
+The [MINI2 Receiver](http://www.femtomes.com) is connected to a UART on the flight controller (GPS port) for data.
+To power the module you will need a separate 12V power supply 
+The pins on the 12-pin connector are numbered as shown below
+
+<img src="http://www.femtomes.com/image/PX4/MINI_II_2.jpg" width="300px" title="port" />
+
+## Configuration
+
+For heading estimation the two antennas need to be on the same level and at least 30 cm apart from each other.
+The direction that they are facing does not matter as it can be configured with the [GPS_YAW_OFFSET](../advanced_config/parameter_reference.md#GPS_YAW_OFFSET) parameter.
+
+Configure the serial port on which the [MINI2 Receiver](http://www.femtomes.com)  will run using [GPS_1_CONFIG](../advanced_config/parameter_reference.md#GPS_1_CONFIG), and set the baud rate to 115200 using [SER_GPS1_BAUD](../advanced_config/parameter_reference.md#SER_GPS1_BAUD).
+
+[Built in FB672](http://www.femtomes.com)

```

Comment:
Everything below here does not look like it is part of a section called "Configuration". Is this some kind of additional specification?
Perhaps put a heading above here. 

```
## Additional Information

The MINI2 incorporates the following components.